Production/test/condorSub/dict_RunIISummer16MiniAODv2_background.py

```diff
@@ -0,0 +1,26 @@
+flist = {
+    "scenario": "Summer16",
+    "samples": [
```

Comment:
need to add TTJets inclusive in this list...

Reply:
Done